core/blockly.js

```diff
@@ -132,6 +132,12 @@ Blockly.cache3dSupported_ = null;
  */
 Blockly.theme_ = null;
 
+/**
+ * Blockly core version.
+ * @define {string}
```

Comment:
What's `define`?  Did you mean `type`?
https://google.github.io/styleguide/jsguide.html#appendices-jsdoc-tag-reference

Reply:
Nope, it's handy for replacing that value for local compilations. (although not using it now, it seemed like the right usecase for it). see https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#define-type-description

Reply:
Oh, neat.  Is there a reason not to use that feature by passing a flag to the compiler rather than editing the code?

Reply:
Unfortunately the online compiler doesn't support it :(

Reply:
Sounds like a good GTI project.  :)